platforms/jvm/testing-jvm/src/main/java/org/gradle/api/tasks/testing/TestReport.java

```diff
@@ -104,6 +111,14 @@ public void setDestinationDir(File destinationDir) {
      * @since 7.4
      */
     @OutputDirectory
+    @UpgradedProperty(
+        originalAccessors = {
+            @UpgradedAccessor(value = GETTER, methodName = "getDestinationDir"),
+            @UpgradedAccessor(value = SETTER, methodName = "setDestinationDir")
+        },
```

Comment:
:thinking: WDYT about renaming `methodName` to `name`, like:

```suggestion
        originalAccessors = {
            @UpgradedAccessor(value = GETTER, name = "getDestinationDir"),
            @UpgradedAccessor(value = SETTER, name = "setDestinationDir")
        },
```

I think it's sufficiently non-ambiguous this way, too. If we did that, we could:

```suggestion
        originalAccessors = {
            @UpgradedAccessor(value = PROPERTY, name = "destinationDir")
        },
```

Probably not worth it...

Reply:
Done